pppd/plugins/radius/dict.c

```diff
@@ -150,7 +150,7 @@ int rc_read_dictionary (char *filename)
 			{
 				type = PW_TYPE_INTEGER;
 			}
-			else if (strcmp (typestr, "ipaddr") == 0)
+			else if (strcmp (typestr, "ipaddr") == 0 || strcmp (typestr, "ipv4addr") == 0)
```

Comment:
I think it makes sense to have this change as it's own PR to be honest.  This is good to go and makes zero sense to have this blocked whilst sorting out the rest of the PR.